Dockerfile.prod

```diff
@@ -20,6 +20,5 @@ COPY --from=build /usr/src/app/package-lock.json ./package-lock.json
 COPY --from=build /usr/src/app/build build/
 RUN npm install --omit=dev
 
-ENV PORT=80
```

Comment:
The idea is to leave the PORT env var to be defined in the server?

Reply:
That is the idea, but if it doesn't matter which port is chosen, we can have it as it was before.